### PR TITLE
Display appropriate information for Welcome Talk and Lunch Break in Timetable detail screen

### LINKED ISSUE #213: Display appropriate information for Welcome Talk and Lunch Break in Timetable detail screen
## Overview
Currently, when tapping on Welcome Talk or Lunch Break in the Timetable screen and transitioning to the detail screen, it displays the event date, time, location, and the text "Special". We need to update this to show more appropriate information for these special session types.

## Links
- Design: https://www.figma.com/design/XUk8WMbKCeIdWD5cz9P9JC/DroidKaigi-2024-App-UI
- Code: https://github.com/DroidKaigi/conference-app-2024/blob/ec7c35cc7a425acaf2d86bdd9d48780268c9f5ea/feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/TimetableItemDetailScreen.kt#L81

### PR DESCRIPTION
**Pull request by hiraike32**
## Issue
- close #213 

## Overview (Required)
- Changed the details screen for special content such as Welcome Talk and Lunch Break to display the appropriate wording.

## Links
- nothing

## Screenshot (Optional if screenshot test is present or unrelated to UI)
Before | After
:--: | :--:
Screenshot_20240811_173130 | Screenshot_20240811_173037

### CODE REVIEW

File: feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/component/TimetableItemDetailContent.kt [hunk 1]
```diff
@@ -79,7 +79,18 @@ fun TimetableItemDetailContent(
             }
 
             is Special -> {
-                Text(stringResource(SessionsRes.string.special))
+                DescriptionSection(
```
Comment (hiraike32) [inline feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/component/TimetableItemDetailContent.kt:82]: The content is displayed in the same way as in Session. 
Please let me know if there is anything missing or incomplete.

**Review by takahirom — APPROVED** [review]
Looks great! Thanks!

File: feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/component/TimetableItemDetailContent.kt (hunk 1 above)
Comment (takahirom) [reply feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/component/TimetableItemDetailContent.kt:81]: Thank you for letting me know. I think it is ok

**Review by takahirom — APPROVED** [review]
Thank you for your contribution!

**Pull request merged**